recipe/meta.yaml

```diff
@@ -17,18 +17,20 @@ build:
 requirements:
   build:
     - autoconf 2.*
-    - m4 1.4.*
     - perl 5.20.3.1
-    - readline 7.0
-    - openssl 1.0.*
-    - ncurses 5.9
-    - zlib 1.2.11
-    - toolchain
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - m4 1.4.*
```

Comment:
@CJ-Wright, `m4` is usually a build-time dependency just like `autoconf`

Reply:
We can probably relax these pinnings to.